interface/src/app/gene/search/search.js

```diff
@@ -264,18 +264,21 @@ angular.module('adage.gene.search', [
           for (var gene in $scope.selectedGenes) {
```

Comment:
I mean for the cases in which we are not looping over an array. I know that for an array we should use the good old ``for (var i = 0; i < this.length; i++) {``.